integrations/telegram/src/index.ts

```diff
@@ -149,19 +154,22 @@ const integration = new bp.Integration({
       throw new Error('Handler received an empty chat id')
     }
 
+    const userId = data.message.from?.id
+    const chatId = data.message.chat?.id
+
+    if (!userId) {
+      throw new Error('Handler received an empty from id')
+    }
+
     const { conversation } = await client.getOrCreateConversation({
       channel: 'channel',
       tags: {
         [idTag]: `${conversationId}`,
+        [senderIdTag]: `${userId}`,
```

Comment:
nitpick 🙂 

```ts
        [idTag]: conversationId,
        [senderIdTag]: userId,
        [chatIdTag]: chatId,
```

Reply:
these are supposed to be integers so we better convert them to strings no ?